docs/editing-components.md

```diff
@@ -0,0 +1,408 @@
+this file should live in `cfgov/unprocessed/css/<atoms|molecules|organisms>/`.
+(Choose the deepest folder according to the atomic rank of the component.)
+Continuing the `RelatedContent` example, if it needed its own styles,
+it would live at `cfgov/unprocessed/css/molecules/related-content.less`.
+
+Newly-created Less files need to be imported into the project's master
+`main.less` file, located at `cfgov/unprocessed/css/main.less`.
+Please place them in the appropriate section for their atomic rank.
+
+Because cfgov-refresh uses `main.less` to build a single CSS file
+for almost the entire project, it is not necessary
+to tell the Python model anything about a component-specific stylesheet
+(for general-purpose, site-wide components).
+That is _not_ the case with JavaScript, as we will see in the next section.
+
+!!! note
+    If you're working on a component that belongs to a particular sub-app,
+    its Less file should live in `cfgov/unprocessed/<app-name>/css/`,
+    but how those Less files get built and included on their pages is
+    not something this document is prepared to discuss at the moment.
+
+#### Adding some JavaScript
+
+Each atomic component may optionally be given a `Media` class that can
+list one or more JavaScript files that should be loaded when using it.
+When a page is requested via the browser,
+[code contained in `base.html`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/jinja2/v1/_layouts/base.html#L100-L110)
+will loop all atomic components for the requested page and
+load the appropriate atomic JavaScript bundles.
+
+Here is how one would add the `Media` class to our `RelatedContent` example:
+
+```python
+class RelatedContent(blocks.StructBlock):
+    …  # see first example on this page
+
+    class Media:
+        js = ['related-content.js']
+```
+
+(The `related-content.js` file would need to be placed in
+`cfgov/unprocessed/js/molecules/`.)
+
+This will load the `related-content.js` script on any page
+that includes the `RelatedContent` molecule in one of its StreamFields.
+
+
+
+
+## How-to guides
+
+
+### Adding a field
+
+1. Locate the Python class of the component you want to add a field to.
+1. Add the field by inserting a snippet like this in the list of fields,
+   in the order in which you want it to appear in the editor:
+   `field_name = blocks.BlockName()`.
+   - Replace `field_name` with a succinct name for what data the field contains
+   - Replace `BlockName` with one of the
+     [basic Wagtail block types](https://docs.wagtail.io/en/v1.13.4/topics/streamfield.html#basic-block-types).
+     Sometimes we create our own custom blocks that can be used, as well.
+     See, for example, the
+     [`HeadingBlock`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/blocks.py#L147-L165),
+     [used in `InfoUnitGroup`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/atomic_elements/organisms.py#L54),
+     among other places.
+1. Add any desired parameters:
```

Comment:
Are you planning to fix the nested lists issue that shows up here (and elsewhere on the page) where the Markdown is just adding `<li>`'s without adding the `<ul>` for the nested list? 
![screen shot 2018-12-14 at 11 59 32 am](https://user-images.githubusercontent.com/702526/50016711-f166f080-ff97-11e8-99a0-fa6e4ac6c54c.png)

Reply:
Huh. Honestly, I never ran it locally to take a look. TIL that the MD renderer that MkDocs uses requires 4-space indentation for nested lists. Fixed in 89f23454e89e7befda7ec3cbe69e2b4ca61cd59e